src/kernel/arch/aarch64/tty.zig

```diff
@@ -2,90 +2,117 @@ const std = @import("std");
 const arch = @import("arch.zig");
 const TTY = @import("../../tty.zig").TTY;
 const panic = @import("../../panic.zig").panic;
-const log = @import("../../log.zig");
+const log = std.log.scoped(.aarch64_tty);
 const mailbox = @import("mailbox.zig");
+const mem = @import("../../mem.zig");
 const Tag = mailbox.Tag;
 
-const CHAR_WIDTH: u8 = 8;
-const CHAR_HEIGHT: u8 = 8;
+pub const CHAR_WIDTH: u8 = 8;
+pub const CHAR_HEIGHT: u8 = 8;
 const BLACK = Pixel{ .red = 0, .blue = 0, .green = 0 };
 const WHITE = Pixel{ .red = 255, .blue = 255, .green = 255 };
```

Comment:
We have are constants just above the `var` and the the public constants below the private ones. See https://github.com/ZystemOS/pluto/wiki/Code-Styling-(Detailed)

Reply:
Done